scripts/effects/dot_acid.script

```diff
@@ -0,0 +1,61 @@
+//Thothie
+
+//ACID
+//Any npc
+//PARAM1 == Duration
+//PARAM2 == Damge Source (entity)
+//PARAM3 == Damage per second
+//PARAM5 == XP Skill
+
+// . effectmestack effects/dot_acid 10 GAME_MASTER 0.5 none
+
+#scope server
+
+{
+	const EFFECT_ID "DOT_acid"
+
+	const EFFECT_SCRIPT $currentscript //Must be copied, as is, at the top script!
+
+	const DOT_TYPE acid_effect
+	
+	const DOT_IM_AFFECTED "You have been acid burned!" //Player recieves affect and is afflicted
+	
+	const DOT_IM_RESIST "You resist the acid attack." //Player recieves the affect, but resists / immune it.
+	const DOT_HE_IMMUNE "is immune to acid!" //Target recieves the affect, but resists / immune it
+}
+
+#include effects/base_dot	allowduplicate
+
+{ dot_start
+
+	local DUR_RATIO $get(PARAM2,scriptvar,'PLR_CORRODE_DURATION')
+	if ( DUR_RATIO != 0 )
+	{
+		multiply EFFECT_DURATION DUR_RATIO //Increases duration
+	}
+
+	if ( $get(ent_me,isplayer) ) svplaysound game.sound.body game.sound.maxvol $get(ent_me,scriptvar,'PLR_SOUND_STOMACHHIT1')
+
+	effect glow ent_me (75,215,0) 72 EFFECT_DURATION EFFECT_DURATION
+	
+	applyeffect ent_me effects/debuff_acid EFFECT_DURATION
+	
+	hud.addstatusicon ent_me hud/status/alpha_dot_poison EFFECT_ID EFFECT_DURATION
+}
+
+{ dot_effect
+
+	//screenfade <target/"all"> <duration> <holdtime> <(r,g,b)> <alpha> <fadein|fadeout|noblend|perm>
+	effect screenfade ent_me 0.2 0 (75,215,0) 30 fadein
+}
+
+{ game_damaged
+
+	//take 150% damage while under this effect
```

Comment:
This applies the acid x1.5 effect and as well as the `debuff_acid` effect that does another x1.5 multiplier.

I was messing around with some acid stuff in this branch by total chance and noticed that it wasn't doing the expected damage lol

Reply:
Oh, thats not intended. Thank you for letting me know